### PR TITLE
Fix touch emulation in the recent Chrome (closes #2448)

### LINKED ISSUE #2448: Unable to detect touchable device
### Are you requesting a feature or reporting a bug?
thats a good question, probably reporting a bug

### What is the current behavior?
There seems to be no way to correctly detect a touchable device when using chrome mobile emulation

### What is the expected behavior?

```
'ontouchstart' in window
```

returns true when chrome mobile emulator is used

### How would you reproduce the current behavior (if this is a bug)?

```
  console.log(`touch? ${await testcafe.eval(() => {
    return 'ontouchstart' in window;
  })}`);
```

run testcafe with chrome mobile emulation (tested with iphone X)

### Specify your

* operating system:
tested on osx 10.13.12 and ubuntu 16.04
* testcafe version:
0.19.2
* node.js version:
8.9.1
* chrome version:
66.0.3359.139

Hello there,

our application is detecting touch devices to run different javascript code for the page initialization. We want to test that the touchable version is working fine using testcafe. In order to detect a touch device, we use the following logic:

```
((("ontouchstart" in window ||
                        !!(window.DocumentTouch && document instanceof DocumentTouch) ||
                        window.navigator.maxTouchPoints)) && 
                        !window._phantom && !window.phantom) ? true : false;
```

This works fine in the chrome devtools for example. Select a mobile device, reload the page and executing the code will return true.
However, when we use chrome mobile emulation provided by testcafe, the code returns false and there seem to be 0 registered touch event listener in the document.

Is there a better way to detect touch devices or is this an issue with testcafe mobile emulation?

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexKamaev

### CODE REVIEW

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
Let's add test for this

File: test/functional/fixtures/browser-provider/chrome-emulation/testcafe-fixtures/index-test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `Touch events`
+    .page `example.com`;
```
Comment (miherlosev) [inline test/functional/fixtures/browser-provider/chrome-emulation/testcafe-fixtures/index-test.js]: I guess it's necessary. You can use the `about:blank` page.

File: test/functional/fixtures/browser-provider/chrome-emulation/testcafe-fixtures/index-test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `Touch events`
+    .page `about:blank`;
```
Comment (miherlosev) [inline test/functional/fixtures/browser-provider/chrome-emulation/testcafe-fixtures/index-test.js]: Remove the `page` directive completely:

```js
fixture `Touch events`;
```

**Pull request merged**